x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -992,3 +898,226 @@ export const SHOWING_EXCEPTION_LISTS = (totalLists: number) =>
+    }
+  );
+
+export const RULES_BULK_ENABLE_FAILURE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.enable.errorToastTitle',
+  {
+    defaultMessage: 'Error enabling rules',
+  }
+);
+
+export const RULES_BULK_ENABLE_FAILURE_DESCRIPTION = (rulesCount: number) =>
+  i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.enable.errorToastDescription',
+    {
+      values: { rulesCount },
+      defaultMessage: '{rulesCount, plural, =1 {# rule is} other {# rules are}} failed to enable.',
+    }
+  );
+
+/**
+ * Bulk Disable
+ */
+
+export const RULES_BULK_DISABLE_SUCCESS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.disable.successToastTitle',
+  {
+    defaultMessage: 'Rules disabled',
+  }
+);
+
+export const RULES_BULK_DISABLE_SUCCESS_DESCRIPTION = (totalRules: number) =>
+  i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.disable.successToastDescription',
+    {
+      values: { totalRules },
+      defaultMessage:
+        'Successfully disabled {totalRules, plural, =1 {{totalRules} rule} other {{totalRules} rules}}',
+    }
+  );
+
+export const RULES_BULK_DISABLE_FAILURE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.disable.errorToastTitle',
+  {
+    defaultMessage: 'Error disabling rules',
+  }
+);
+
+export const RULES_BULK_DISABLE_FAILURE_DESCRIPTION = (rulesCount: number) =>
+  i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.disable.errorToastDescription',
+    {
+      values: { rulesCount },
+      defaultMessage: '{rulesCount, plural, =1 {# rule is} other {# rules are}} failed to disable.',
+    }
+  );
+
+/**
+ * Bulk Edit
+ */
+
+export const RULES_BULK_EDIT_SUCCESS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.edit.successToastTitle',
+  {
+    defaultMessage: 'Rules changes updated',
+  }
+);
+
+export const RULES_BULK_EDIT_SUCCESS_DESCRIPTION = (rulesCount: number) =>
+  i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.edit.successToastDescription',
+    {
+      values: { rulesCount },
+      defaultMessage:
+        "You've successfully updated {rulesCount, plural, =1 {# rule} other {# rules}}.",
+    }
+  );
+
+export const RULES_BULK_EDIT_FAILURE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.edit.errorToastTitle',
+  {
+    defaultMessage: 'Error updating rules',
+  }
+);
+
+export const RULES_BULK_EDIT_FAILURE_DESCRIPTION = (rulesCount: number) =>
+  i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.edit.errorToastDescription',
+    {
+      values: { rulesCount },
+      defaultMessage: '{rulesCount, plural, =1 {# rule is} other {# rules are}} failed to update.',
```

Comment:
Perhaps `{X} rules failed to update` for failed toast description?
<img width="365" alt="image" src="https://user-images.githubusercontent.com/2946766/159617851-b35ef79a-5efa-4073-9a7a-f5eb25d5ac81.png">


```suggestion
      defaultMessage: '{rulesCount, plural, =1 {# rule} other {# rules}} failed to update.',
```